src/nvim/optionstr.c

```diff
@@ -696,8 +696,11 @@ const char *did_set_buftype(optset_T *args)
       || opt_strings_flags(buf->b_p_bt, opt_bt_values, NULL, false) != OK) {
     return e_invarg;
   }
-  // buftype=prompt: set the prompt start position to lastline.
+  // buftype=prompt:
   if (buf->b_p_bt[0] == 'p') {
+    // Set default value for 'comments'
```

Comment:
mention in docs ?

Reply:
I'm not sure where to. Also, most of the filetypes set this option those aren't really documented in docs is it?

Reply:
'buftype' is much more important than the numerous filetypes.

anyway, i'll document this in my docs pr